From 12ce6fe529e690fc6e705a62d610679eb6b130f7 Mon Sep 17 00:00:00 2001
From: jdeolive <Joao.DeOliveiraSilva@windriver.com>
Date: Wed, 7 Feb 2024 13:45:11 -0300
Subject: [PATCH] Removing unnecessary 0

Removes the zero from the data pool from cephfs name to better match 
the current baremetal ceph usage on Wind River systems.

Signed-off-by: jdeolive <Joao.DeOliveiraSilva@windriver.com>
---
 cluster/charts/rook-ceph-cluster/templates/cephfilesystem.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cluster/charts/rook-ceph-cluster/templates/cephfilesystem.yaml b/cluster/charts/rook-ceph-cluster/templates/cephfilesystem.yaml
index d2dc1cd12..57d6c2f1a 100644
--- a/cluster/charts/rook-ceph-cluster/templates/cephfilesystem.yaml
+++ b/cluster/charts/rook-ceph-cluster/templates/cephfilesystem.yaml
@@ -18,7 +18,7 @@ metadata:
 provisioner: {{ $root.Values.operatorNamespace }}.cephfs.csi.ceph.com
 parameters:
   fsName: {{ $filesystem.name }}
-  pool: {{ $filesystem.name }}-data0
+  pool: {{ $filesystem.name }}-data
   clusterID: {{ $root.Release.Namespace }}
 {{ toYaml $filesystem.storageClass.parameters | indent 2 }}
 reclaimPolicy: {{ default "Delete" $filesystem.storageClass.reclaimPolicy }}
-- 
2.25.1

